desktop/menus/view-menu.js

```diff
@@ -1,84 +1,84 @@
-var buildRadioGroup = function(activePredicate) {
-  return function(item) {
-    var label = item[0],
-      prop = item[1],
-      action = item[2];
+const { buildRadioGroup, appCommandSender } = require('./utils');
 
-    return {
-      label: label,
-      type: 'radio',
-      checked: activePredicate(prop),
-      click: function(item, focusedWindow) {
-        if (!focusedWindow) {
-          return;
-        }
-
-        focusedWindow.webContents.send('appCommand', action);
-      },
-    };
-  };
-};
-
-var buildFontGroup = function(item) {
-  var label = item[0],
-    accelerator = item[1],
-    action = item[2];
-
-  return {
-    label: label,
-    accelerator: accelerator,
-    click: function(item, focusedWindow) {
-      if (!focusedWindow) {
-        return;
-      }
-
-      focusedWindow.webContents.send('appCommand', { action: action });
-    },
-  };
-};
-
-var equalTo = function(a) {
-  return function(b) {
-    return a === b;
-  };
-};
-
-var buildViewMenu = function(settings) {
+const buildViewMenu = settings => {
   settings = settings || {};
 
   return {
     label: '&View',
     submenu: [
       {
-        label: '&Font Size',
+        label: '&Note Display',
+        submenu: [
+          {
+            label: '&Comfy',
+            id: 'comfy',
+          },
+          {
+            label: 'C&ondensed',
+            id: 'condensed',
+          },
+          {
+            label: '&Expanded',
+            id: 'expanded',
+          },
+        ].map(
+          buildRadioGroup({
+            action: 'setNoteDisplay',
+            propName: 'noteDisplay',
+            settings,
+          })
+        ),
+      },
+      {
+        label: 'Note &Editor',
         submenu: [
-          // For the oddity with "Command" vs "Cmd"
-          // Cite: https://github.com/atom/electron/issues/1507
-          ['&Bigger', 'CommandOrControl+=', 'increaseFontSize'],
-          ['&Smaller', 'CommandOrControl+-', 'decreaseFontSize'],
-          ['&Reset', 'CommandOrControl+0', 'resetFontSize'],
-        ].map(buildFontGroup),
+          {
+            label: '&Font Size',
+            submenu: [
+              // For the oddity with "Command" vs "Cmd"
+              // Cite: https://github.com/atom/electron/issues/1507
+              {
+                label: '&Bigger',
+                accelerator: 'CommandOrControl+=',
+                click: appCommandSender({ action: 'increaseFontSize' }),
+              },
+              {
+                label: '&Smaller',
```

Comment:
I wonder how you localize these... 🤔

Reply:
Yeah, it would be difficult... Maybe drop access key support completely, or manually add them to just the top-level menu labels.